index.js

```diff
@@ -1635,8 +1635,22 @@ export async function createJavaBom(path, options) {
         );
         const splitPropTaskOut =
           splitOutputByGradleProjects(parallelPropTaskOut);
+
         for (const [key, propTaskOut] of splitPropTaskOut.entries()) {
-          const retMap = parseGradleProperties(propTaskOut);
+          let retMap = {};
+          // To optimize performance and reduce errors do not query for properties
+          // beyond the first level. Replicating behaviour from single-threaded Gradle generation.
+          if (key.includes(":")) {
```

Comment:
Wouldn't the first level keys not start with colon?

Reply:
They won't, I add the key into the map after removing the first colon. So if it still contains any more colons, that would indicate some level of nesting.